lib/web/apiserver.go

```diff
@@ -2865,6 +2868,43 @@ func (h *Handler) getClusters(w http.ResponseWriter, r *http.Request, p httprout
 	return out, nil
 }
 
+type GetClusterInfoResponse struct {
```

Comment:
nit: this does not need to be an exported struct and  since you are only using this to respond from `getClusterInfo`, I think it would be better for this one to be tucked inside the `getClusterInfo`.